helm/scylla-manager/templates/manager_deployment.yaml

```diff
@@ -38,6 +38,13 @@ spec:
         volumeMounts:
         - mountPath: /mnt/etc/scylla-manager
           name: scylla-manager-config
+        readinessProbe:
+          exec:
```

Comment:
execs are expensive, can we use the manager server api? like 5080/version or 5080/clusters?

Reply:
changed to GET :5080/api/v1/clusters